packages/variants/src/variants.ts

```diff
@@ -264,6 +266,7 @@ export function getStrongVariant(theme: IPartialTheme): ITheme {
     buttonBackground: p.white,
     buttonBackgroundHovered: !fullTheme.isInverted ? p.themeLighter : p.themeLight,
     buttonBackgroundPressed: !fullTheme.isInverted ? p.themeLight : p.themeTertiary,
+    buttonBackgroundDisabled: p.themePrimary,
```

Comment:
shouldn't this be a shade off of themePrimary?

Reply:
No we have a simplified schema where both button styles are just the same.

Reply:
Checked in with the designer, it looks like the values the designer gave me were for fluent. She's getting me updated info for pre-fluent styles now

Reply:
@bennettclark yeah make it separate. Make sure design knows that live code needs to be non-fluent, and fluent stuff needs to go through Eddie's flighting mechanism.